README.md

```diff
@@ -583,6 +583,25 @@ process.on('uncaughtException', async () => {
   process.exit(1);
 });
 ```
+#### Queue#isRunning()
+
+Returns `true` unless the Queue is shutting down due to a call to `Queue#close()`;
+
+#### Queue#ready([cb])
+
+Promise resolves to the queue (or callback is called wth `null` argument) when the queue (and Redis) are ready for jobs.  Learn more about `'ready'` in [Queue Local Events](#queue-local-events).
+
+```js
+const Queue = require('bee-queue');
+const queue = new Queue('example');
+queue.ready()
+  .then(async queue => {
```

Comment:
for consistency
```suggestion
  .then(async (queue) => {
```

Reply:
Stylistic habit from my current long-term project...